test/cache/AccountLinkCacheSpec.scala

```diff
@@ -36,8 +36,7 @@ class AccountLinkCacheSpec
   private val test: DutyDefermentAccountLink =
     DutyDefermentAccountLink("someEori", "dan", "linkId", AccountStatusOpen, DefermentAccountAvailable, isNiAccount)
 
-  private val app       = application().build()
-  private val testCache = app.injector.instanceOf[AccountLinkCache]
+  private val testCache = application().injector.instanceOf[AccountLinkCache]
```

Comment:
Said the same in Jamie's PR. Would be a nice-to-have `def instanceOf[???] = application().injector.instanceOf[???]` in `SpecBase` but U2Y.